test/spec/functional/viewActivation.js

```diff
@@ -0,0 +1,242 @@
+    require('./clientEnv')().done(function(environment) {
+      env = environment;
+      $ = env.window.$;
+      _ = env.window._;
+      View = env.window.Torso.Views.View;
+      ClickView = require(testSrcPath + '/ClickView')(View, _, spyOnBackbone);
+      $('body').append('<div class="app"></div>');
+      done();
+    });
+  });
+
+  /**
+   * Creates a child view class with different dom
+   */
+  var createChildView = function() {
+    return require(testSrcPath + '/ChildClickView')(View, _, spyOnBackbone);
+  };
+
+  /**
+   * Creates a parent view with two children views
+   */
+  var setUpParentView = function(ChildView1, ChildView2) {
+    return require(testSrcPath + '/ParentClickView')(View, _, spyOnBackbone, ChildView1, ChildView2);
+  };
+
+
+  it('can be iniatialized correctly', function() {
+    var view = new ClickView();
+    expect(view.$el).toBeDefined();
+    view.attach($('div.app'));
+    expect(view.myClick).not.toHaveBeenCalled();
+    view.$el.find('div').click().change();
+    expect(view.myClick).toHaveBeenCalled();
+    expect(view.afterMyEvent).not.toHaveBeenCalled();
+    view.trigger('myEvent');
+    expect(view.afterMyEvent).toHaveBeenCalled();
+    view.dispose();
+  });
+
+  it('can be deactivated correctly', function() {
+    var view = new ClickView();
+    expect(view.$el).toBeDefined();
+    view.attach($('div.app'));
+    view.deactivate();
+    expect($('div.app').length).toBe(0);
+    expect($('div.click').length).toBe(1);
+    expect(view.$el).toBeDefined();
+    $('div.app').append('<div>Dont trigger event!</div>');
+    expect(view.myClick).not.toHaveBeenCalled();
+    $('div').click().change();
+    expect(view.myClick).not.toHaveBeenCalled();
+    expect(view.afterMyEvent).not.toHaveBeenCalled();
+    view.trigger('myEvent');
+    expect(view.afterMyEvent).toHaveBeenCalled();
+    expect(view.afterMyDeactivatableEvent).not.toHaveBeenCalled();
+    view.trigger('myDeactivatableEvent');
+    expect(view.afterMyDeactivatableEvent).not.toHaveBeenCalled();
+    view.dispose();
+  });
+
+  it('can be activated correctly', function() {
+    var view = new ClickView();
+    expect(view.$el).toBeDefined();
+    view.attach($('div.app'));
+    view.deactivate();
+    view.activate();
+    expect($('div.app').length).toBe(0);
+    expect($('div.click').length).toBe(1);
+    expect(view.myClick).not.toHaveBeenCalled();
+    view.$el.find('div').click().change();
+    expect(view.myClick).toHaveBeenCalled();
+    expect(view.afterMyEvent).not.toHaveBeenCalled();
+    view.trigger('myEvent');
+    expect(view.afterMyEvent).toHaveBeenCalled();
+    expect(view.afterMyDeactivatableEvent).not.toHaveBeenCalled();
+    view.trigger('myDeactivatableEvent');
+    expect(view.afterMyDeactivatableEvent).toHaveBeenCalled();
+    view.dispose();
+  });
+
+  it('can be initialized correctly with child views', function() {
+    var ParentView = setUpParentView(createChildView(), createChildView());
```

Comment:
that comment doesn't make sense.  This is weird I'm trying to think of how to make it clearer.